<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLC模拟通信</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>通信日志</string>
    </property>
    <widget class="QTextBrowser" name="comLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>串口参数</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>21</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>串口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="portName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>数据位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dataBits">
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>波特率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="baudRate">
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>停止位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stopBits">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>校验位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="parity">
          <item>
           <property name="text">
            <string>NoParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EvenParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OddParity</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>123</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="openSerialButton">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(255, 244, 253);</string>
          </property>
          <property name="text">
           <string>打开串口</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>发送</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QTextEdit" name="message"/>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>填充异常：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="abnDataButton"/>
        </item>
        <item>
         <widget class="QComboBox" name="abnormalButton"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>清空日志窗口</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnSend">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
